### PR TITLE
Update Epinio installation on Rancher Desktop

### PR DESCRIPTION
**Pull request by ldevulder**

### CODE REVIEW

File: src/installation/install_epinio_on_rancher_desktop.md [hunk 1]
```diff
@@ -19,8 +21,20 @@ Make sure Rancher Desktop is running.
...
+
+The Epinio installation is pretty much identical on Linux, MacOS and Windows:
+1. Start a terminal, use `cmd` or `powershell` on Windows (latest one is preferred) and your preferred one on Linux/MacOS (`bash` is the default one).
 
-### Linux
+2. Download the [latest version](https://github.com/epinio/epinio/releases) of Epinio binary and copy it on your client. It is recommended to add it to the `PATH` variable (`export PATH=<epinio-binary-path>:$PATH` on Linux/MacOS with Bash and on Windows please follow [Kevin Berg's article](https://medium.com/@kevinmarkvi/how-to-add-executables-to-your-path-in-windows-5ffa4ce61a53)).
 
-Linux is not yet supported by Rancher Desktop - see https://github.com/rancher-sandbox/rancher-desktop/issues/426
+3. Use `helm` to install Epinio by following either the [Method 1](./install_epinio_auto.md) (recommended for most cases) or the [Method 2](./install_epinio_manual.md) (useful for advanced configuration or if you already have some components in your environment). As **Traefik** is already installed, just bypass this step (by adding option `--set skipTraefik=true` for Method 1).
```
Comment (juadk) [inline src/installation/install_epinio_on_rancher_desktop.md:40]: I would mention the current issue we have on Linux until it will be fixed, WDYT?

**Review by juadk — APPROVED** [review]
Overall, LGTM, just two small comments.

File: src/installation/install_epinio_on_rancher_desktop.md (hunk 1 above)
Comment (ldevulder) [reply src/installation/install_epinio_on_rancher_desktop.md:40]: Done!

**Comment by juadk** [comment]
Fix #42

**Pull request merged**